explorer/src/entity/JobRun.ts

```diff
@@ -103,9 +103,16 @@ const searchBuilder = (
   params: ISearchParams
 ): SelectQueryBuilder<JobRun> => {
   let query = db.getRepository(JobRun).createQueryBuilder('job_run')
-
   if (params.searchQuery != null) {
-    const searchTokens = params.searchQuery.split(/\s+/)
+    let searchTokens = params.searchQuery.split(/\s+/)
```

Comment:
I made a refactor in #1201 to extract search into it's own area which will conflict with this. I'd like to wait until that's merged and block this PR. You could rebase on that branch if you want to keep working on it.

I'd also like to see test coverage around this feature before merging.